vizro-core/docs/pages/development/documentation-style-guide.md

```diff
@@ -0,0 +1,143 @@
+
+## Vizro lexicon
+
+The names of our products are **Vizro** and **Vizro-AI**.
+
+We refer to other products using their preferred capitalization. For example, Dash and Plotly are always capitalized, except where given as Python package names `dash` and `plotly`.
+
+Vizro components are named using lower case:
+
+> "Here is a guide to using containers..."
+
+Use code font when referring to the component as a class or object:
+
+> "To add a `Container` to your page..."
+
+## Bullets
+* Capitalize the first word, and end the bullet with a period.
+* Don't use numbered bullets except for a sequence of instructions, or where you have to refer back to one of them in the text (or a diagram).
+
+## Call out boxes
+
+Keep the amount of text, and the number and variety of callouts used, to a minimum. There is a [broad set available](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) for use in the Vizro docs, but we prefer to limit usage mostly to the following:
+
+!!!note "note"
+
+     For notable information
+
+!!!warning "warning"
+
+     To indicate a potential gotcha.
+
+!!!example "example"
+
+     For example code
+
+
+Boxes can be made collapsible, and if you choose this option, prefer to add them to the page so they are initially collapsed, to avoid crowding.
+
+## Capitalization
+* Only capitalize proper nouns e.g. names of technology products, other tools and services.
+* Don't capitalize cloud, internet, machine learning, or advanced analytics. Take a look at the [Microsoft style guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/accessibility-terms) if you're unsure.
+* Follow sentence case, which capitalizes only the first word of a title/subtitle. We prefer this "An introduction to data visualization" to this "An Introduction to Data Visualization".
+
+
+## Code formatting
+* Mark code blocks with the appropriate language to enable syntax highlighting.
+* We use a `bash` lexer for all codeblocks that represent the terminal, and we don't include the prompt.
+* Use the code format for Python package names such as `pandas` or `pylint`.
+
+## Instructions
+
+Prefer to use imperatives to make instructions. For example:
+> "Complete the configuration steps"
+
+You don't need to use the word "please" -- readers want less to read and don't think it's rude if you omit it.
+
+You can also use second person:
+> "You should complete the configuration steps". Don't use the passive "The configuration steps should be completed" (see next bullet).
```

Comment:
This looks strangely formatted  - should the "don't use the passive" be in here? Also not sure what "see next bullet" refers to?